server/api-manager.js

```diff
@@ -1,7 +1,7 @@
 /*eslint-disable comma-spacing */
 /*eslint-disable no-console*/
 
-import {appSettings} from "./config/reports.js"
+import fs from "fs";
```

Comment:
To load the config file, I reused code from Madison and Jason's branches. This should minimize merge conflicts when all three branches go back into dev